docs/sources/static/set-up/install/install-agent-linux.md

```diff
@@ -72,7 +72,6 @@ To install Grafana Agent in static mode on RHEL or Fedora, run the following com
 1. Create `/etc/yum.repos.d/grafana.repo` with the following content:
 
    ```shell
```

Comment:
```suggestion
   ```ini
```

Reply:
@aam1r  I merged before I saw your comment. Help me understand the reasoning for using `ini` vs `shell` (or `bash`) for the codeblock language here

Reply:
My apologies for the lack of explanation.

.repo files use a similar syntax as INI files (see https://en.wikipedia.org/wiki/INI_file). `shell` is more appropriate for shell commands such as the ones below, whereas `ini` is more appropriate for this particular configuration change.

**Before (`shell`)**:

```shell
[grafana]
name=grafana
baseurl=https://rpm.grafana.com
repo_gpgcheck=1
enabled=1
gpgcheck=1
gpgkey=https://rpm.grafana.com/gpg.key
sslverify=1
sslcacert=/etc/pki/tls/certs/ca-bundle.crt
```

**After (`ini`)**:

```ini
[grafana]
name=grafana
baseurl=https://rpm.grafana.com
repo_gpgcheck=1
enabled=1
gpgcheck=1
gpgkey=https://rpm.grafana.com/gpg.key
sslverify=1
sslcacert=/etc/pki/tls/certs/ca-bundle.crt
```

Notice the better color formatting when using `ini` 🙂 FYI, this is just a cosmetic change with no impact on the overall instructions.

Reply:
Ah cool. For consistency, if I change it here, I'd want to dig through the other places it could also be used. I've got some changes pending/planned for the install docs. I'll give it a try and see how it fits into the existing docs.